be/src/connector/hive_connector.cpp

```diff
@@ -259,7 +259,25 @@ Status HiveDataSource::_init_scanner(RuntimeState* state) {
 
     HdfsScanner* scanner = nullptr;
     auto format = scan_range.file_format;
-    if (dynamic_cast<const HudiTableDescriptor*>(_tuple_desc->table_desc()) && scan_range.hudi_mor_table) {
+
+    bool use_hudi_jni_reader = false;
```

Comment:
Is it better to move this logic from be to fe to simplify be

Reply:
done